src/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/roi_align.hpp

```diff
@@ -37,4 +42,32 @@ class ROIAlignLayerTest : public testing::WithParamInterface<roialignParams>,
     std::string poolingMode;
 };
 
+using roialignV9Params = std::tuple<std::vector<size_t>,             // feature map shape
+                                    std::vector<size_t>,             // proposal coords shape
+                                    int,                             // bin's row count
+                                    int,                             // bin's column count
+                                    float,                           // spatial scale
+                                    int,                             // pooling ratio
+                                    std::string,                     // pooling mode
+                                    std::string,                     // roi aligned mode
```

Comment:
vice versa, introducing one more duplicating enum is an error prone since we have to support the same notion in different places. using specific string names of enums guaranties us of using correct values and gives us loose coupling. we tie string names with enum values by using EnumNames facility that guaranties correct enum names otherwise it throws an exception

Reply:
as for pooling mode, it is already used by cpu tests. aligned mode was added in the same way.
we use here EnumNames facility that guaranties correct conversion of strings to enum names otherwise it throws an exception.
I would leave it as string values